Makefile

```diff
@@ -28,8 +35,8 @@ $(OUT): $(shell find resources -type f)
 # reimports (delete+import) the application into the locally running portal, for development
 #
 reimport: $(OUT)
-	-gravity app --state-dir=$(LOCAL_WORK_DIR) delete $(PACKAGE) --force
-	gravity app --state-dir=$(LOCAL_WORK_DIR) import $(OUT)
+	-gravity app delete $(PACKAGE) $(DELETE_OPTS)
+	gravity app import $(OUT) $(IMPORT_OPTS)
```

Comment:
This changes need for proper importing of package with images. Without vendoring we will got error
```
1h        4m        17    {kubelet 192.168.122.176}    spec.containers{bootstrap}    Normal        Pulling        pulling image "stolon-bootstrap:0.0.1"
  1h        4m        17    {kubelet 192.168.122.176}    spec.containers{bootstrap}    Warning        Failed        Failed to pull image "stolon-bootstrap:0.0.1": Error: image library/stolon-bootstrap:0.0.1 not found
```